Cargo.toml

```
@@ -35,29 +35,30 @@ default = ["bastion", "iouring"]
 epoll = []
 iouring = ["iou", "uring-sys"]
 
-bastion = ["agnostik/runtime_bastion"]
-tokio = ["agnostik/runtime_tokio"]
 asyncstd = ["agnostik/runtime_asyncstd"]
+bastion = ["agnostik/runtime_bastion"]
 smol = ["agnostik/runtime_smol"]
+tokio = ["agnostik/runtime_tokio"]
 
 [dependencies]
 lever = "0.1.1-alpha.8"
 futures = "0.3.5"
 futures-util = "0.3.5"
-socket2 = { version = "0.3.12", features = ["pair", "unix"] }
 libc = "0.2"
-pin-utils = "0.1.0"
 once_cell = "1.4.0"
 agnostik = "0.1.3"
+pin-utils = "0.1.0"
+socket2 = {version = "0.3.12", features = ["pair", "unix"]}
+winapi = "0.3.9"
```

```suggestion
winapi = { version = "0.3.9", default-features = false } 
```

Then add each feature as needed, that will drastically reduce compile times.